mbed_app.json

```diff
@@ -1,8 +1,8 @@
 {
-    "macros": ["MBED_SYS_STATS_ENABLED"],
     "target_overrides": {
         "*": {
-            "platform.stdio-convert-newlines": 1
+            "platform.stdio-convert-newlines": 1,
```

Comment:
Do we explain anywhere why we have this line?

Reply:
I removed it.